CMakeLists.txt

```diff
@@ -7,9 +7,7 @@ include(GNUInstallDirs)
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
-set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
-
-find_package(USD REQUIRED)
+find_package(USD REQUIRED NAMES pxr)
```

Comment:
Interesting ! Could we improve it to `find_package(pxr REQUIRED)` ?

Reply:
Seems to work! Also does not require `-Dpxr_dir` anymore. But: expects USD and its dependencies to be in the PATH (which should be the case for a proper USD installation).